.vscode/launch.json

```diff
@@ -62,15 +62,18 @@
       "preLaunchTask": "CompileGrammarTests"
     },
     {
-      "name": "Run Functional Tests",
+      "name": "Run Functional Test",
```

Comment:
Shouldn't this be a new launch setting? We still want to be able to run all functional tests in VSCode.

Reply:
I was debating that too. Do we actually want to be able to run them all from VSCode or is it ok to just have command line do it given that's how the CI will do it anyhow?

Reply:
Hmm. So if I enter `*` in the prompt, it will run all the tests? If so, that is good enough for me.

Reply:
I think `*` on its own would barf but if you did `*.test.js` it would work.

Can definitely special case `*` though to make that work out of the box.

Reply:
Or can set it as the default value of the prompt.

Reply:
Oooo I like that much better!